lib/node_modules/@stdlib/utils/parse-ndjson/examples/index.js

```diff
@@ -0,0 +1,64 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+var parseNDJSON = require( './../lib' );
+
+var ndjsonString = '{"name":"John"}\n{"name":"Doe"}';
+
+// Example 1: Parse a simple NDJSON string
+var parsedArray = parseNDJSON(ndjsonString);
```

Comment:
```suggestion
var parsedArray = parseNDJSON( ndjsonString );
```

Spacing here and below.